metadata-collector/main.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/NVIDIA/go-nvml/pkg/nvml"
+)
+
+func main() {
+	log.Println("Starting GPU Metadata Collector (Hello World)")
```

Comment:
Can we bootstrap and use the common logging like we have in https://github.com/NVIDIA/NVSentinel/blob/main/health-monitors/syslog-health-monitor/main.go#L73-L74

Reply:
using the same now, ptal